doc/app-METHOD.md

```diff
@@ -0,0 +1,80 @@
+<!---
+ Copyright (c) 2016 StrongLoop, IBM, and Express Contributors
+ License: MIT
+-->
+
+<h3 id='app.METHOD'>app.METHOD(path, callback [, callback ...])</h3>
+
+Routes an HTTP request, where METHOD is the HTTP method of the request, such as GET,
+PUT, POST, and so on, in lowercase. Thus, the actual methods are `app.get()`,
+`app.post()`, `app.put()`, and so on.  See below for the complete list.
+
+For more information, see the [routing guide](/guide/routing.html).
```

Comment:
If that is the stance, I don' even see what the point of putting the documentation in this repository is. Just keep them in the website repository, which is the whole point of why they were never in here in the first place. To move them in here, means we need to make it work correctly, otherwise there is no purpose, because nothing keeps people from making pull requests to the website repository.

Ideally, since we need to maintain jsdoc comments for tools like WebStorm, all this is now duplicating all that content anyway. Why don't we just generate the documentation from the jsdoc comments and forget duplicating everything three times?